IFC4x3/Properties/i/IsReducedResistanceFastening_0DOivAfc56c81gaVUiqBi8/Documentation.md

```diff
@@ -0,0 +1 @@
+Is Reduced Resistance Fastening
```

Comment:
Maybe definition can be improved? "True if..."

Reply:
I have improved it. Commit : c8245c45a7f488800e3e5c5dd32f498fffa28154